Exebite.DataAccess/Context/MealOrderingContext.cs

```diff
@@ -85,12 +87,12 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<MealToCondimentEntity>()
                 .HasKey(k => new { k.MealId, k.CondimentId });
 
-            modelBuilder.Entity<CustomerToFavouriteMealEntity>()
-                .HasKey(k => new { k.CustomerId, k.MealId });
-
             modelBuilder.Entity<CommentEntity>()
                 .HasKey(k => new { k.MealId, k.CommentId });
 
```

Comment:
Add following code as discussed in #192 

```suggestion
            modelBuilder.Entity<CommentEntity>()
                .Property(c => c.CommentId)
                .ValueGeneratedOnAdd();

            modelBuilder.Entity<CommentEntity>()
                .Property(c => c.NumberOfLikes)
                .HasDefaultValue(default(long))
                .ValueGeneratedOnAdd();
```

Reply:
Change was made inside #192